src/app.js

```diff
@@ -109,4 +109,4 @@ for (let controller of controllers) {
 
 // app.get('/', (req, res) => res.send('Hello World!'))
 
-app.listen(port, () => console.log(`Example app listening at http://localhost:${port}`))
\ No newline at end of file
+app.listen(port, () => console.log(`Example app listening at http://localhost:${port}`));
```

Comment:
the semicolon you added is not so much needed because JavaScript don't really mind